package.json

```diff
@@ -51,6 +51,10 @@
     "typescript": "^3.9.5",
     "webpack": "^4.26.0",
     "webpack-cli": "^3.1.2",
-    "webpack-dev-server": "^3.1.10"
+    "webpack-dev-server": "^3.1.10",
+    "@webgpu/glslang": "0.0.12",
+    "@webgpu/types": "0.0.24",
+    "@tensorflow/tfjs-backend-webgpu": "0.0.1-alpha.4",
```

Comment:
@BruceDai Does it mean @lionkunonly has added WebGPU backend support successfully?

Reply:
@lionkunonly We've already added WebGPU backend support on [Line#30](https://github.com/intel/webml-polyfill/blob/master/package.json#L30), [Line#32](https://github.com/intel/webml-polyfill/blob/master/package.json#L32), [Line#33](https://github.com/intel/webml-polyfill/blob/master/package.json#L33), please remove these duplicated devDependencies, thanks.

Reply:
Done